en/sensor/rangefinders.md

```diff
@@ -78,16 +78,27 @@ The uLanding Radar is not present in "most" firmware by default and must be star
 The *LeddarOne* must typically be added to firmware before it can be used. For setup/usage information see: [LeddarOne](../sensor/leddar_one.md).
 
 
-### TFmini 
+### TFmini
 
-The [Benewake TFmini LiDAR](http://www.benewake.com/en/tfmini.html) is a tiny, low cost, and low power LIDAR with 12m range. 
-
-The sensor should be connected to the SERIAL 4/5 port on Pixhawk devices. It is enabled by setting parameter [SENS_EN_TFMINI](../advanced_config/parameter_reference.md#SENS_EN_TFMINI) > 0.
+The [Benewake TFmini LiDAR](http://www.benewake.com/en/tfmini.html) is a tiny, low cost, and low power LIDAR with 12m range.
 
 > **Note** This lidar is currently in master, and should appear in releases from PX4 v1.8.

```

Comment:
Is this still correct? This isn't going into any 1.7 builds?

Reply:
Not sure what the release plans are... I guess it will be in the next one. Is the next one 1.7.x?

Reply:
Probably. I think I'll leave this until I see another release and check then.

Reply:
Ok, thanks!